api/src/pcapi/core/subscription/dms/api.py

```diff
@@ -1,5 +1,4 @@
 import datetime
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Commit message 2 en 1 ?

Reply:
ah oui ^^